Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_19_0.md

```diff
@@ -0,0 +1,21 @@
+
+#### Scripts
+
+##### New: MicrosoftAtpIsolateMachine
+
+New: Added a new ***MicrosoftAtpIsolateMachine*** script that isolates a machine from accessing external networks.
+
+
+##### New: MicrosoftAtpUnisolateMachine
+
+New: Added a new ***MicrosoftAtpUnisolateMachine*** script that reverses the isolation of a machine.
+
+
+##### New: MicrosoftAtpEndpoint
+
+New: Added a new ***MicrosoftAtpEndpoint*** script that gets machines that have communicated with Microsoft Defender for Endpoint cloud.
+
+
+##### New: MicrosoftAtpScIndicatorCreate
+
+New: Added a new ***MicrosoftAtpScIndicatorCreate*** script that creates a new indicator in Microsoft Defender for Endpoint.
```

Comment:
Added the **MicrosoftAtpScIndicatorCreate** script that creates a new indicator in Microsoft Defender for Endpoint.